### PR TITLE
feat(base): Add initial postcard creation app

### PR DESCRIPTION
**Pull request by pdufour**
# what
add angular bootcamp code so that you can create a new postcard

# testing instructions
- run the server
- navigate to http://localhost:8888/#!/create and see that you can create a new postcard

### CODE REVIEW

**Review by shraddha1704 — APPROVED** [review]

File: app/create/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var Angular = require('angular');
+
+var Controller = require('./controller');
+var State = require('./state');
```
Comment (lvamaral) [inline app/create/index.js]: nit: align these `require`s

File: app/index.js [hunk 2]
```diff
@@ -9,11 +9,13 @@ var Config   = require('./config');
 var Home     = require('./home');
 var Run      = require('./run');
 var Services = require('./services');
+var Create = require('./create');
```
Comment (lvamaral) [inline app/index.js]: nit: align + we typically try doing ABC order for the declarations

File: app/services/index.js [hunk 3]
```diff
@@ -6,11 +6,13 @@ var Cookies = require('angular-cookies');
 var API     = require('./api');
 var Auth    = require('./auth');
 var Session = require('./session');
+var Postcard = require('./postcard');
```
Comment (lvamaral) [inline app/services/index.js:7]: nit: align

File: test/create/controller.spec.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+var expect  = require('chai').expect;
+var Angular = require('angular');
+var Sinon   = require('sinon');
+
+require('angular-mocks');
```
Comment (lvamaral) [inline test/create/controller.spec.js:7]: the onboarding docs might be outdated, but just leaving this here for info, but we've moved away from using `mocks` in our angular service (Dashboard) in favor of `stub`s

**Review by lvamaral — APPROVED** [review]

**Comment by lvamaral** [comment]
lgtm, just some styling nits!

**Pull request merged**